### ISSUE TITLE
Allow empty alt text on the Image component

### ISSUE THREAD

**Issue by blittle** [Shopify/hydrogen-ui#37]
**Describe the bug**
The `Image` component currently throws warnings when there is no alt text. Sometimes, for presentational images, there should not be alt text. Should an explicit empty string passed as a prop avoid the warning? cc @WillsonSmith
